src/components/FeedCard.tsx

```diff
@@ -199,13 +199,14 @@ export function TitleAndBylinePlaceholder({creator}: {creator?: boolean}) {
 }
 
 export function Description({description}: {description?: string}) {
-  const [rt, isResolving] = useRichText(description || '')
-  if (!description) return null
-  return isResolving ? (
-    <RichText value={description} style={[a.leading_snug]} />
-  ) : (
-    <RichText value={rt} style={[a.leading_snug]} />
-  )
+  const [rt] = React.useState(() => {
```

Comment:
i think this should be a useMemo? or it won’t be reactive

Reply:
Sure might as well 👍